changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.81.2
+ * Closing realm on another thread different from which it was created on throws an exception.
```

Comment:
Closing a Realm on a thread different from which it was created now throws an exception.